opentech/apply/funds/models.py

```diff
@@ -767,6 +767,9 @@ def can_review(self, user):
 
         return self.has_permission_to_review(user)
 
+    def user_lead_or_admin(self, user):
+        return user.is_superuser or self.lead.id == user.id
```

Comment:
Can't say i'm a massive fan. can we keep it descriptive like the `has_permission_to_review` methods?

also `self.lead == user` is sufficient

Reply:
changed to `has_permission_to_add_determination`, yet I am not a fan of that as it couples things even further